verticapy/performance/vertica/collection/profile_import.py

```diff
@@ -141,11 +210,92 @@ def _create_schema_if_not_exists(self) -> None:
 
     def _create_tables_if_not_exists(self) -> None:
         all_tables = getAllCollectionTables(
-            target_schema=self.target_schema, key=self.key
+            target_schema=self.target_schema, key=self.key, version=self.bundle_version
```

Comment:
The same concern here when `self.bundle_version` is None.

Reply:
There are several other similar cases as well. Is it a desired behavior for these function to fail if they are called prior to the `check_file` function?

Reply:
For testing purposes, I want to be able to call `check_schema` and `check_file` separately. 

I will refactor with a centralized way to set version if it has not been set.

Reply:
I have refactored this so that `self.bundle_version` is `LATEST` by default, instead of unset. That is sufficient for the kinds of tests we want to run.